src/FormattingSettingsComponents.ts

```diff
@@ -19,31 +19,97 @@ class NamedEntity {
 }
 
 export class Model {
-    cards: Array<Card>;
+    cards: Array<Card | CompositeCard>;
 }
 
-export class Card extends NamedEntity implements IFormattingSettingsCard {
+export abstract class CompositeCard extends NamedEntity {
```

Comment:
please add support for topLevelToggle in CompositeCard, not only in its groups

Reply:
Isn't it that `topLevelToggle` only exist at the level of Slice and Group? And that the first slice/group that has the `topLevelToggle` attribute set to `true` 'promoted' to the parent level either Card or Group?

Reply:
I have removed the `topLevelToggle` attributed from both the `SimpleCard` and `CompositeCard` classes.
The current implementation is based on the current implementation: the `ToggleSwitch` with the property `topLevelToggle` set to `true` from the first group is used as the card `topLevelToggle`. In all the other groups the `ToggleSwitch` with the property `topLevelToggle` set to `true` is used as the group `topLevelToggle`.

![image](https://user-images.githubusercontent.com/12474072/227546278-631c187e-89d7-4f23-bfbf-61c63ad04f0d.png)
`CompositeCard` with two groups with each a `ToggleSwitch` slice with `topLevelToggle = true`

Reply:
Its not a good idea because:
1. its not clear and missleading, since the user will put toplevelToggler=true in first group and suddenly when he run the CV he will see that the topLevelToggle appears in the card.
2. What if the user want the first group with topLevelToggle and not the card?
3. What if the user want the composite card with topLevelToggle? then how should he know that first group topLevelToggle will appear in the card
4. If the customer have composite card with multiple groups and decideds to change groups order then this will affect his CV topLevelToggle

Reply:
Make sense. I will add `topLevelToggle` attribute as a single slice to `Group`, 'SimpleCard` and `CompositeCard`.
Can I also remove the `topLevelToggle` attribute from the `ToggleSwitch` class? https://github.com/microsoft/powerbi-visuals-utils-formattingmodel/blob/e14c2dac3fa0d6ba16107b22c388fbc304dbb1d7/src/FormattingSettingsComponents.ts#L125

This will also introduce a breaking change with the current version as that version doesn't have a dedicated `topLevelToggle` attribute as the `Card` class and use the inherited behavior.

Reply:
@MulyukovAidar @AleksSavelev

Reply:
Sorry, I lost track on this issue, but @uve can we continue in raising to 6.0.0 and breaking the backwards combability?